nautobot/docs/development/core/settings.md

```diff
@@ -85,3 +85,20 @@ Markdown rendering is supported for the `description`, `details`, and `default_l
 ### Technical Details of Settings Documentation
 
 The `optional-settings.md` and `required-settings.md` files are rendered as Jinja2 templates via [`Mkdocs-Macros`](https://mkdocs-macros-plugin.readthedocs.io/en/latest/). The file `nautobot/docs/macros.py` is responsible for loading `settings.yaml` into the template context for rendering. `mkdocs.yml` instructs Mkdocs-Macros to run that file at documentation rendering time. The macros and templating are *not* enabled for all documentation by default - instead, only the files with `render_macros: true` in their headers will be templated.
+
+## Structlog Configuration
+
+As of Nautobot 2.3, structlog is added as a core dependency. To use structlog, you can add the following code to your `nautobot_config.py` file:
```

Comment:
```suggestion
As of Nautobot 2.3, [structlog](https://www.structlog.org/en/stable/) and [django-structlog](https://django-structlog.readthedocs.io/en/latest/) are added as core dependencies; however they are not enabled by default at this time. To use structlog, you can add the following code to your `nautobot_config.py` file:
```

Reply:
Actually, only `django-structlog` is added as a dependecny, fixed.